src/components/Card/Card.jsx

```diff
@@ -47,34 +68,37 @@ export const CardHeader = (
 export const CardTitle = (
   { children, title } //eslint-disable-line
 ) => (
-  <span className="card--title" title={title}>
+  <span className={`${iotPrefix}--card--title`} title={title}>
     {children}
   </span>
 );
 
-export const CardContent = styled.div`
-  flex: 1;
-  position: relative;
-  height: ${props => props.dimensions.y - CARD_TITLE_HEIGHT}px;
-  overflow-x: visible;
-  overflow-y: ${props => (!props.isExpanded ? 'visible' : 'auto')};
-`;
-
-export const SkeletonWrapper = styled.div`
-  padding: ${CARD_CONTENT_PADDING}px;
-  width: 80%;
-`;
+const CardContent = props => {
+  const { children, dimensions, isExpanded } = props;
+  const height = `${dimensions.y - CARD_TITLE_HEIGHT}px`;
+  return (
+    <div
+      style={{ [`--card-content-height`]: height }}
```

Comment:
Should this be its own class?

Reply:
Do you mean an HTML class for the height? The height is a JS calculated value so that is why it is set as a custom property (CSS variable) and not prop in a class like the other fixed values.

Reply:
Ah I see. I read that as a boolean 👍